yin_yang/plugins/system.py

```diff
@@ -48,6 +50,41 @@ def available(self) -> bool:
         return test_gnome_availability(self.command)
 
 
+class _Budgie(PluginCommandline):
+    name = 'System'
+
+    def __init__(self):
+        super().__init__(['gsettings', 'set', 'com.solus-project.budgie-panel', 'dark-theme', '{theme}'])
+        self.theme_light = 'light'
+        self.theme_dark = 'dark'
+
+    @property
+    def available(self) -> bool:
+        return test_gnome_availability(self.command)
+
+    '''Override because budgie uses a switch for dark/light mode'''
```

Comment:
Please use a comment, or put the docstring after the method declaration with a blank line afterwards